runtimevar/runtimevar.go

```diff
@@ -479,6 +480,27 @@ func bytesDecode(b []byte, obj interface{}) error {
 	return nil
 }
 
+// DecryptDecode returns a decode function that can be passed to NewDecoder when
+// decoding an encrypted message (https://godoc.org/gocloud.dev/secrets).
+//
+// The decode function by default will decode the input into raw bytes. An
```

Comment:
Maybe we should export `stringDecode` and `bytesDecode`? Then this can say `post defaults to `BytesDecode`.

Reply:
Done.